Dockerfile

```diff
@@ -3,19 +3,19 @@ ARG UBI_VERSION=9
 ARG OPENJDK_VERSION=17
 ARG UBI_BUILD_TAG=latest
 ARG UBI_RUNTIME_TAG=latest
-ARG IMAGE_BUILD=registry.access.redhat.com/ubi${UBI_VERSION}/openjdk-${OPENJDK_VERSION}:${UBI_TAG}
+ARG IMAGE_BUILD=registry.access.redhat.com/ubi${UBI_VERSION}/openjdk-${OPENJDK_VERSION}:${UBI_BUILD_TAG}
 ARG IMAGE_BASE=registry.access.redhat.com/ubi${UBI_VERSION}/openjdk-${OPENJDK_VERSION}-runtime:${UBI_RUNTIME_TAG}
 
 #DEV FROM $IMAGE_BUILD AS build
-FROM registry.access.redhat.com/ubi9/openjdk-17:1.15-1.1686736679 AS build
+FROM ${IMAGE_BUILD} AS build
```

Comment:
Unless @enkeys wants to argue for not changing this (in which case I'll solidarity support him), I don't personally mind making this change.

+1